modules/pca_webform/src/Element/WebformAddressLoqate.php

```diff
@@ -60,43 +60,45 @@ public static function preRenderWebformCompositeFormElement($element) {
    */
   public static function getCompositeElements(array $element) {
 
+    $t = \Drupal::translation();
```

Comment:
$translation or $drupalTranslation instead of $t ?

Reply:
Thanks, I will use `$drupal_translation` (snake case) for local variables